kano_updater/scenarios.py

```diff
@@ -278,6 +279,11 @@ def beta_132_to_beta_133(self):
 
     def beta_133_to_beta_134(self):
         pass
-    
+
     def beta_134_to_beta_200(self):
-        pass
+        if is_installed('kano-character-cli'):
+            cmd = 'kano-character-cli -c "Judoka_Base" "Hair_Black" "Skin_Orange" -r -s'
+            run_cmd_log(cmd)
+        else:
+            logger.warn(
```

Comment:
@alex5imon I think that the role of the check is that if the update fails really badly, this won't make it worse. If you don't have the `kano-profile` installed that usually means that your install is pretty badly damaged.